ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -327,6 +327,30 @@ EOF
   echo "Enabled AWS Spot instances for worker nodes"
 }
 
+# enable_efa_pg_instance_config is an AWS specific option that enables one worker machineset in a placement group and with EFA Network Interface Type, other worker machinesets will be ENA Network Interface Type by default.....
+function enable_efa_pg_instance_config() {
+  local dir=${1}
+
+  if [ ! -f /tmp/yq ]; then
+    curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+  fi
+
+  PATCH="${SHARED_DIR}/machineset-efa-pg-instance.yaml.patch"
+  cat > "${PATCH}" << EOF
+spec:
+  template:
+    spec:
+      providerSpec:
+        value:
+          networkInterfaceType: EFA
+          instanceType: c5n.9xlarge
+          placementGroupName: pgcluster  
+EOF
+
+  /tmp/yq m -x -i "$dir/openshift/99_openshift-cluster-api_worker-machineset-0.yaml" "${PATCH}"
```

Comment:
only patch one machineset?

Reply:
> only patch one machineset?

Yes, this will be https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/blob/master/functionality-testing/aos-4_14/ipi-on-aws/versioned-installer-ovn-efa-ena-ci, it only changes one worker machineset with EFA Network Interface Type, other worker machinesets will be ENA Network Interface Type by default, because such instance type is too large, for example c5n.9xlarge, 36 vCPU, 96GiB Memory, and we think one worker EFA should be ok for the testing.